core/src/main/java/com/sequenceiq/cloudbreak/service/events/DefaultCloudbreakEventService.java

```diff
@@ -121,18 +121,27 @@ public void fireCloudbreakInstanceGroupEvent(Long stackId, String eventType, Str
     public List<StructuredNotificationEvent> cloudbreakEventsForStack(Long stackId) {
         List<StructuredNotificationEvent> events = new ArrayList<>();
         if (stackId != null) {
-            events = structuredEventService.getEventsWithTypeAndResourceId(StructuredNotificationEvent.class, "stacks", stackId);
+            events = structuredEventService.getEventsWithTypeAndResourceId(StructuredNotificationEvent.class,
+                    CloudbreakEventService.DATAHUB_RESOURCE_TYPE, stackId);
```

Comment:
is it always datahub? I think the same is called for datalake

Reply:
wip